Packs/Core/Playbooks/playbook-SSO_Brute_Force.yml

```diff
@@ -0,0 +1,1051 @@
+    id: "22"
+    taskid: d766566f-458b-49b1-87dc-59bfc76ce5ca
+    type: regular
+    task:
+      id: d766566f-458b-49b1-87dc-59bfc76ce5ca
+      version: -1
+      name: Get user MFA configuration to assess their resilience
+      description: Returns all the enrolled factors for the specified user.
+      script: '|||okta-get-user-factors'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    scriptarguments:
+      extend-context:
+        simple: MFAResult=
+      username:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: auth_identity
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 850,
+          "y": 620
+        }
+      }
+    note: true
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 44da91f4-abdd-4773-819e-fadd023d4614
+    type: collection
+    task:
+      id: 44da91f4-abdd-4773-819e-fadd023d4614
+      version: -1
+      name: Decide if to expire the user password
+      type: collection
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "25"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1730,
+          "y": 1620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Would you like to expire the password of the user ${alert.username.[0]} using ${Core.OriginalAlert.event.vendor}?
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - {}
+        - simple: "Yes"
+        - simple: "No"
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Select to which user you like to expire password to
+      description: Decide whether the password should be expired for the user.
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: be42e18c-391c-4925-8eb2-c2984464dca2
+    type: condition
+    task:
+      id: be42e18c-391c-4925-8eb2-c2984464dca2
+      version: -1
+      name: Should enforce password expire for the user?
```

Comment:
Same about the English @ShirleyDenkberg @ArikDay 
Think that's better:
Should enforce password expiration for the user?
WDYT?